cmd/helm/list.go

```diff
@@ -96,7 +96,7 @@ func newListCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 	f.BoolVarP(&client.Short, "short", "q", false, "output short (quiet) listing format")
 	f.BoolVarP(&client.ByDate, "date", "d", false, "sort by release date")
 	f.BoolVarP(&client.SortReverse, "reverse", "r", false, "reverse the sort order")
-	f.BoolVarP(&client.All, "all", "a", false, "show all releases, not just the ones marked deployed or failed")
+	f.BoolVarP(&client.All, "all", "a", false, "show all releases in specified namespace(default namespace by default), not just the ones marked deployed or failed")
```

Comment:
"show all releases in specified namespace(default namespace by default), not just the ones marked deployed or failed" --> "show all releases in specified namespace (uses current namespace context otherwise), not just the ones marked deployed or failed"